packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -109,7 +110,9 @@ export class CdkErrorMapper {
       classification: 'ERROR',
     },
     {
-      errorRegex: /(SyntaxError|ReferenceError|TypeError):((?:.|\n)*?at .*)/,
+      errorRegex: new RegExp(
+        `(SyntaxError|ReferenceError|TypeError):((?:.|${EOL})*?at .*)`
```

Comment:
an option to consider here https://stackoverflow.com/questions/1331815/regular-expression-to-match-cross-platform-newline-characters .

in case CDK starts to emit just `\n` on windows some day.

Reply:
Fair point, though that would be a disaster of unimaginable proportion. I'd keep OS specific handling for now, if cdk does something unimaginable, worse we would see an increase in our telemetry at which point we can revisit this.

Reply:
Looks like this is already a reality, CDK doesn't seem to generate OS specific line endings already 🤦‍♂️ 🥹 👊

Reply:
lol :-)